nextflow_schema.json

```diff
@@ -321,8 +321,7 @@
             "default": 15
         },
         "save_preprocessed_reads": {
-            "type": "boolean",
-            "default": false
+            "type": "boolean"
```

Comment:
Why remove the default?

Reply:
I'm not sure what has/how that happened here... will revert that

Reply:
Ah, ok I thnk _default_ for boolean IS false. But the nf-core schema uses `False` to indicate this. 

I guess when you load the build interface it'll just pull what ever is in the config by default. But the build-schema interface removes it (to match the actual schema correctly) on save?

If you look all the other boolean values they don't have any default.

Reply:
I see, that makes sense. Thanks 🙂